libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.kt

```diff
@@ -531,6 +547,20 @@ open class SalesforceSDKManager protected constructor(
      */
     var shouldBlockSalesforceIntegrationUser = false
 
+    /**
+     * Creates a NativeLoginManager instance that allows the app to use it's
```

Comment:
Simply typo: This should be possessive `its` instead of a contraction for `it is`.